tests/Helpers/HelperTest.php

```diff
@@ -60,3 +61,123 @@
 test('getFormEditPageUrl returns the correct url.', function () {
 	$this->assertSame('/wp-admin/post.php?post=1&action=edit', Helper::getFormEditPageUrl('1'));
 });
+
+//---------------------------------------------------------------------------------//
+
+test('getFormTrashActionUrl calls get_delete_post_link with correct arguments.', function () {
+	$this->assertSame('id: 0, force: false', Helper::getFormTrashActionUrl('0'));
+	$this->assertSame('id: 0, force: false', Helper::getFormTrashActionUrl('0', 0));
+	$this->assertSame('id: 0, force: true', Helper::getFormTrashActionUrl('0', true));
+	$this->assertSame('id: 0, force: true', Helper::getFormTrashActionUrl('0', 1));
+	$this->assertSame('id: 75, force: true', Helper::getFormTrashActionUrl('75', 1));
+});
```

Comment:
You could use expectations API coupled with a datasets for similar returning cases.

https://pestphp.com/docs/datasets
https://pestphp.com/docs/expectations

The test would then be something like

```php
test('getFormTrashActionUrl calls get_delete_post_link with correct arguments.', function ($data) {
	expect(Helper::getFormTrashActionUrl($data))
		->toBe('id: 0, force: false');
})->with([
    ['0'],
    ['0', 0]
]);;
```

Or without using datasets, you can just put multiple expectations (they look nicer imo).